sensor/extrinsic_reflector_based_calibrator/scripts/metrics_plotter_node.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python3
+
+# Copyright 2023 Tier IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+
+import matplotlib.pyplot as plt
+import rclpy
+from rclpy.node import Node
+from std_msgs.msg import Float32MultiArray
+
+
+class Plotter:
+    def __init__(self):
+        self.fig, self.axes = plt.subplots(nrows=2, ncols=2, figsize=(8, 6))
+        self.subplot0 = self.axes[0, 0]
+        self.subplot1 = self.axes[0, 1]
+        self.subplot2 = self.axes[1, 0]
+        self.subplot3 = self.axes[1, 1]
+        plt.gcf().canvas.set_window_title("Metrics plotter")
+
+        self.color_bo = "bo-"
+        self.color_go = "go-"
+        self.color_b = "b"
+        self.color_g = "g"
+
+        (
+            self.num_of_reflectors,
+            self.cv_distance_error,
+            self.cv_yaw_error,
+            self.calibration_distance_error,
+            self.calibration_yaw_error,
+        ) = (0, 0, 0, 0, 0)
+        (
+            self.prev_calibration_distance_error,
+            self.prev_calibration_yaw_error,
+            self.prev_cv_distance_error,
+            self.prev_cv_yaw_error,
+            self.prev_num_of_reflectors,
+        ) = (0, 0, 0, 0, 0)
+        (
+            self.cv_distance_error_list,
+            self.cv_yaw_error_list,
+            self.calibration_distance_error_list,
+            self.calibration_yaw_error_list,
+            self.num_of_reflectors_list,
+        ) = ([0], [0], [0], [0], [0])
+        self.anno0, self.anno1, self.anno2, self.anno3 = None, None, None, None
+        self.max_ylim0, self.max_ylim1, self.max_ylim2, self.max_ylim3 = 0, 0, 0, 0
+
+        self.init_plot()
+        plt.tight_layout()
+        plt.pause(0.1)
+
+    def init_plot(self):
+        self.subplot0.set_title("Cross-validation error: distance")
+        self.subplot0.set_xlabel("numbers of track")
+        self.subplot0.set_ylabel("Distance error (cm)")
+
+        self.subplot1.set_title("Cross-validation error: yaw")
+        self.subplot1.set_xlabel("numbers of track")
+        self.subplot1.set_ylabel("Yaw error (deg)")
```

Comment:
nit: usually the notation is [cm] instead of (cm) but I will leave it up to you

Reply:
Commit  https://github.com/tier4/CalibrationTools/pull/145/commits/d2299288a6f82602bcc8eead60a504072958ca67 fix this.